openbb_platform/providers/intrinio/openbb_intrinio/models/income_statement.py

```diff
@@ -71,7 +71,7 @@ def extract_data(
         api_key = credentials.get("intrinio_api_key") if credentials else ""
 
         base_url = "https://api-v2.intrinio.com"
-        url_params = f"{query.symbol}-balance_sheet_statement-{query.year}"
+        url_params = f"{query.symbol}-income_statement-{query.year}"
```

Comment:
symbol="JPM" returns:

```
OpenBBError: Intrinio endpoint doesn't exist
```

It does exist though, so the URLs are not generating correctly:

```
https://api-v2.intrinio.com/fundamentals/JPM-income_statement-2023-Q2/standardized_financials?
```